src/dash-table/derived/edges/index.ts

```diff
@@ -14,15 +14,22 @@ function resolveEdges(styles: IConvertedStyle[]): BorderStyle {
             const border = s.style[p] || s.style.border;
 
             if (border) {
-                res[p] = [border, i];
+                res[p] = [border, priority ?? i];
```

Comment:
null-coalescing operator, equivalent to `isNil(priority) ? i : priority`

Reply:
Whoa, I'd never seen that before 🥇 That could simplify our lives in a million places! Can you confirm that it's properly transpiled for [IE and the other browsers that don't support it](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)?

Reply:
Yup - the null-coalescing operator is a new addition to ESx, and supported by TypeScript since 3.7
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing.

This transpiles to (yes, old style `void 0` for undefined!):
```
if (border) {
    res[p] = [border, priority !== null && priority !== void 0 ? priority : i];
}
```